packages/init/src/steps/sendStarterData.ts

```diff
@@ -78,23 +78,34 @@ async function setupS3(
       filename,
     });
 
-    const res = await axios.post(acl.values.url, form, {
-      headers: form.getHeaders(),
-    });
-
-    const s3ImageUrl = `${acl.imgixEndpoint}/${key}`;
-
-    if (res.status !== 204) {
-      const msg =
-        "[slice/push] An error occurred while uploading files - please contact support";
-      console.error(msg);
-      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
-      console.error(`Error code: "${res.status}"`);
-
-      throw new Error(msg);
-    }
-
-    return s3ImageUrl;
+    const errorMessage = `[slice/push] An error occurred while uploading preview images for ${sliceName}-${variationId} - please contact support`;
+
+    return axios
+      .post(acl.values.url, form, {
+        headers: form.getHeaders(),
+      })
+      .then((res) => {
+        if (res.status !== 204) {
```

Comment:
you don't have to check the status, it's already a success.

Reply:
It's the same as the code in slice-machine when sending the images through the acl service in
packages/slice-machine/server/src/api/services/uploadScreenshotClient.ts

I guess there are some 200 codes that can be errors